lib/ast-utils.js

```diff
@@ -24,6 +24,8 @@ const bindOrCallOrApplyPattern = /^(?:bind|call|apply)$/;
 const breakableTypePattern = /^(?:(?:Do)?While|For(?:In|Of)?|Switch)Statement$/;
 const thisTagPattern = /^[\s*]*@this/m;
 
+
+const COMMENTS_IGNORE_PATTERN = "^\\s*(?:eslint|jshint\\s+|jslint\\s+|istanbul\\s+|globals?\\s+|exported\\s+|jscs|falls?\\s?through)";
```

Comment:
I used what was in `line-comment-position` (then found a similar one in `capitalized-comments` and unified that too).

IIRC, the original reason to use a string was for parallelism with the `ignorePattern` option, in code and as a reference so someone could see it and use it as a base. No strong opinion on this, though.